.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+#### What does this PR do?
+#### Any background context you want to provide?
+#### What ticket does this PR close?
+Contributes to [relevant GitHub issues, eg #76]
+#### Where should the reviewer start?
+#### How should this be manually tested?
```

Comment:
I'd like to have a system, just like the `start` `stop` `test` system we use for automated tests.  So the question could even be "Does this include a new folder in the `manual` test dir?", and there would be a detailed README, per our system.

Rather than putting that knowledge in the PR description.